### ISSUE TITLE
Setting "overwrite" to false still gives me Overwriting variable 'xyz'

### ISSUE THREAD

**Issue by mwager** [za-creature/coffeescope#2]
First: Thanks for this, waiting a long time for such a plugin :)

Example:

``` coffeescript
describe 'some spec', ->
  someVar = null

  beforeEach ->
    someVar = 123

```

Output:

⚡ #5: Overwriting variable 'someVar' (first defined on line 2).

Do I miss anything related to the overwrite option?

Docs say:

```
A boolean representing whether to warn when assigning to a
variable that was defined in a parent scope. Because
coffeescript lacks a `let` statement, assigning to a
variable will only create a new variable if there are no
matching variables of the same name in the current scope.
This rule allows you to discourage code that relies on this
and prevents unintentional occurences. The default value is
<samp>true</samp> meaning assigning variables from an outer
scope will issue a warning / error. The special value "all"
will issue a warning if a variable is modified regardless
of the scope it was defined in, effectively preventing
reference mutation. In this regard, it forces all variables
to behave similarly to the <code>const</code> keyword in
ES6+.
```

**Comment by za-creature** [comment]
Fixed in 0.1.3; thanks!

**Comment by mwager** [comment]
Hey that was fast, thanks!
